planner/core/src/main/java/eu/seaclouds/platform/planner/core/Planner.java

```diff
@@ -105,6 +108,7 @@ private String generatMMOutput(Map<String, HashSet<String>> mmResult, Map<String
         return mapper.writerWithDefaultPrettyPrinter().writeValueAsString(singleRes);
     }
 
+
     public String plan() throws ParsingException, IOException {
```

Comment:
In general, avoid strings where other types are more appropriate.

Also why this method doesn't have any parameters?